src/__tests__/CatNew.test.js

```diff
@@ -0,0 +1,45 @@
+import { render, screen } from "@testing-library/react"
+import CatNew from "../pages/CatNew"
+import { BrowserRouter } from "react-router-dom"
+import React from "react"
+
+describe("<CatNew />", () => {
+    it("renders the form", () => {
+        render(
+            <BrowserRouter>
+                <CatNew createCat={() => {}} />
```

Comment:
While CatNew does receive props for the createCat function, you won't need those props for the tests you have written.